src/conn.rs

```diff
@@ -135,6 +139,11 @@ impl<'a, 'c> Queryable for InProgressRead<'a, 'c> {
         self.0.q_once(query, inputs)
     }
 
+    fn q_prepare<T>(&self, query: &str, inputs: T) -> PreparedResult
```

Comment:
How are you getting away without specifying the lifetime on the `PreparedResult`? Magic!

I wonder if the compiler successfully prevents you from leaking the prepared query outside the scope of the ongoing transaction… it should.

Reply:
I had to specify an explicit lifetime after rebasing. 😄